### PR TITLE
Rewrite _currentDate to work in all time zones

### PR DESCRIPTION
**Pull request by amelchio**
I got so annoyed with this bug that I am now attempting my first frontend submission.

**Fixes:** home-assistant/home-assistant#6902

### CODE REVIEW

File: panels/history/ha-panel-history.html [hunk 1]
```diff
@@ -120,9 +120,13 @@
     _currentDate: {
       type: String,
       value: function () {
-        var value = new Date();
-        value.setHours(0, 0, 0);
-        return value.toISOString().split('T')[0];
+        var today = new Date();
```
Comment (armills) [inline panels/history/ha-panel-history.html]: What about this? A bit more concise.

```javascript
var value = new Date();
var today = new Date(Date.UTC(value.getFullYear(), value.getMonth(), value.getDate()))
return today.toISOString().split('T')[0];
```

Comment (amelchio) [inline panels/history/ha-panel-history.html]: I, for one, welcome our new Date object overlord. I have added a commit with this as an alternative.

**Review by emlove — APPROVED** [review]
Looks reasonable to me

**Comment by balloob** [comment]
It's ugly but it works.

**Pull request merged**